oryx-app-mllib/src/test/java/com/cloudera/oryx/app/mllib/kmeans/AbstractKMeansEvalIT.java

```diff
@@ -27,17 +32,17 @@
 import com.cloudera.oryx.app.kmeans.KMeansPMMLUtils;
 import com.cloudera.oryx.app.kmeans.KMeansPMMLUtilsTest;
 import com.cloudera.oryx.app.kmeans.SquaredDistanceFn;
-import com.cloudera.oryx.common.OryxTest;
+import com.cloudera.oryx.lambda.AbstractSparkIT;
 
-public abstract class AbstractKMeansEvalTest extends OryxTest {
```

Comment:
Eh, this is still removing the unit tests right? the unit tests seemed fine. I think we're looking at adding an IT for SparkContext-based tests separately